src/magic.cpp

```diff
@@ -315,6 +317,8 @@ void spell_type::load( const JsonObject &jo, const std::string & )
     spell_class = trait_id( temp_string );
     optional( jo, was_loaded, "energy_source", temp_string, "NONE" );
     energy_source = energy_source_from_string( temp_string );
+    optional( jo, was_loaded, "energy_item", temp_string, "NONE" );
```

Comment:
Should test if `energy_source == item_energy` and use `mandatory` to read the energy source item.

Reply:
Good idea